Orange/widgets/visualize/tests/test_owscatterplot.py

```diff
@@ -591,6 +577,15 @@ def assert_equal(data, max):
         data.Y[42] = np.nan
         assert_equal(data, MAX_CATEGORIES + 1)
 
+    def test_change_data(self):
+        self.send_signal(self.widget.Inputs.data, self.data)
+        self.send_signal(self.widget.Inputs.data, Table("titanic"))
+        self.assertTrue(self.widget.Warning.no_continuous_vars.is_shown())
+        self.assertIsNone(self.widget.data)
```

Comment:
Instead of checking self.widget.data I'd suggest checking output.
`self.assertIsNone(self.get_output(self.widget.Outputs.annotated_data))`.

Reply:
Right. But it doesn't hurt to do both.